### ISSUE TITLE
Precompute embeddings

### ISSUE THREAD

**Issue by isamu-isozaki** [huggingface/open-muse#70]

**Comment by pcuenca** [comment]
Is the idea here to pre-compute the encodings of the dataset with the VQGAN encoder? If so, I've done similar things in the past, could probably help.

**Comment by isamu-isozaki** [comment]
@pcuenca Yup exactly! We are thinking of doing this with t5/VQGAN in the training/train_muse.py. Any help will be awesome!
